tensorflow/python/ops/string_ops.py

```diff
@@ -193,7 +193,7 @@ def string_format(template, inputs, placeholder="{}", summarize=3, name=None):
 
 
 @tf_export("string_split")
-def string_split(source, delimiter=" ", skip_empty=True):  # pylint: disable=invalid-name
+def string_split(source, delimiter=" ", skip_empty=True, maxsplit=-1):  # pylint: disable=invalid-name
```

Comment:
The changes in this file seem unrelated. Adding arguments to public API functions in TF requires an extra review step.